src/pages/home/_testimonial-carousel.js

```diff
@@ -0,0 +1,178 @@
+import React, { Children, useState, useEffect, useRef } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Flex } from 'components/containers'
+import ArrowLeft from 'images/svg/testimonials/arrow-left.svg'
+import ArrowRight from 'images/svg/testimonials/arrow-right.svg'
+import ArrowLeftFade from 'images/svg/testimonials/arrow-left-fade.svg'
+import ArrowRightFade from 'images/svg/testimonials/arrow-right-fade.svg'
+import device from 'themes/device'
+
+const CarouselContainer = styled(Flex)`
+    position: relative;
+
+    .carousel-item {
+        position: relative;
+        overflow: hidden;
+        transition: all 0.3s ease-in;
+        align-items: center;
+
+        .flexible-height {
+            position: relative;
+            transition: opacity 0.3s ease-in;
+
+            .flexi-item {
+                margin-bottom: 0;
+            }
+        }
+    }
+    .height-scanner {
+        opacity: 0;
+    }
```

Comment:
We typically try to avoid class names with the use of styled-components.  You can extend divs or target specific styled-components within the CarouselContainer, for e.g. `${Navigation} { }`.